<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>timeManager</class>
 <widget class="QDockWidget" name="timeManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>199</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>234</width>
    <height>81</height>
   </size>
  </property>
  <property name="floating">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Raster Data Plotting</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QWidget" name="g1" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Plot Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="_plotType">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Scatter / Density Plot</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Spectral Profile Plot</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Temporal Profile Plot</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="_refresh">
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="toolTip">
          <string>Refresh plot.</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/themes/default/mActionRefresh.svg</normaloff>:/images/themes/default/mActionRefresh.svg</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="_stacked">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="_pageUndefined"/>
      <widget class="QWidget" name="_pageScatterPlot">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="RdpScatterPlotWidget" name="_scatterPlot" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="_pageSpectralProfilePlot">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="RdpSpectralProfilePlotWidget" name="_spectralProfilePlot" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="_pageTemporalProfilePlot">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="RdpTemporalProfilePlotWidget" name="_temporalProfilePlot" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RdpScatterPlotWidget</class>
   <extends>QWidget</extends>
   <header>rasterdataplotting.rasterdataplotting.gui.rdpscatterplotwidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RdpSpectralProfilePlotWidget</class>
   <extends>QWidget</extends>
   <header>rasterdataplotting.rasterdataplotting.gui.rdpspectralprofileplotwidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RdpTemporalProfilePlotWidget</class>
   <extends>QWidget</extends>
   <header>rasterdataplotting.rasterdataplotting.gui.rdptemporalprofileplotwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>_plotType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>_stacked</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
